adserver/decisionengine/backends.py

```diff
@@ -273,8 +278,8 @@ def filter_flight(self, flight):
         if not flight.show_on_domain(self.url):
             return False
 
-        # Skip if there are no clicks or views needed today (ad pacing)
-        if flight.weighted_clicks_needed_today() <= 0:
+        # Skip if there are no clicks or views needed today/this interval (ad pacing)
+        if flight.weighted_clicks_needed_today(interval=self.interval) <= 0:
```

Comment:
Seems like passing in `flight.interval` would be simple here? With the model field being defaulted to the setting?